Dockerfile

```
@@ -0,0 +1,31 @@
+# Install system dependencies
+FROM python:3.10-buster AS base
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -
+
+# Copy across dependecy management files
+WORKDIR /todo-app
+COPY poetry.lock /todo-app
+COPY pyproject.toml /todo-app
```

Not important, but worth noting that we can combine these if we want to:
```suggestion
COPY poetry.lock pyproject.toml /todo-app/
```
(The final slash isn't always important, but Docker seems to sometimes moan about it being missing!)